### PR TITLE
Mbes/152 implement i bundle

### PR DESCRIPTION
**Pull request by mbusk88**
## Description
Get market operator data from storage when a bundle already exists in DB and when none exists.

## References

* #152

### CODE REVIEW

File: samples/SubDomain/GetMessage/Storage/StorageController.cs [hunk 1]
```diff
@@ -28,7 +28,7 @@ public StorageController(BlobServiceClient blobServiceClient)
 
         public async Task<System.Uri> CreateBlobRessourceAsync(string blobName, System.BinaryData data)
         {
-            var containerClient = _blobServiceClient.GetBlobContainerClient("getmessage-reply-blobstorage");
+            var containerClient = _blobServiceClient.GetBlobContainerClient("timeseries");
```
Comment (xnetsrak) [inline samples/SubDomain/GetMessage/Storage/StorageController.cs:31]: Make environment variable for container name

**Review by xnetsrak — CHANGES_REQUESTED** [review]
just reviewed sample project, see comment

File: source/Energinet.DataHub.PostOffice.Domain/Services/IMarketOperatorDataStorageService.cs [hunk 2]
```diff
@@ -26,8 +27,9 @@ public interface IMarketOperatorDataStorageService
         /// <summary>
         /// Download data from storage to a market operator.
         /// </summary>
+        /// <param name="bundleUuid">TODO: This is a temporary parameter, see issue #183.</param>
         /// <param name="contentPath">Url to data.</param>
         /// <returns>A Task containing a Stream that contains the data for the market operator.</returns>
-        public Task<Stream> GetMarkedOperatorDataAsync(Uri contentPath);
+        public Task<Stream> GetMarkedOperatorDataAsync(Uuid bundleUuid, Uri contentPath);
```
Comment (mbusk88) [inline source/Energinet.DataHub.PostOffice.Domain/Services/IMarketOperatorDataStorageService.cs:33]: Market with t

File: source/Energinet.DataHub.PostOffice.Tests/Energinet.DataHub.PostOffice.Tests.csproj [hunk 3]
```diff
@@ -34,6 +34,8 @@ limitations under the License.
 
   <ItemGroup>
     <PackageReference Include="AutoFixture" Version="4.17.0" />
```
Comment (michaelmajgaard) [inline source/Energinet.DataHub.PostOffice.Tests/Energinet.DataHub.PostOffice.Tests.csproj:36]: Don't think we use AutoFixture for anything?

**Review by michaelmajgaard — CHANGES_REQUESTED** [review]

**Review by michaelmajgaard — DISMISSED** [review]

File: samples/SubDomain/GetMessage/Storage/StorageController.cs [hunk 4]
```diff
@@ -28,7 +29,7 @@ public StorageController(BlobServiceClient blobServiceClient)
 
         public async Task<System.Uri> CreateBlobRessourceAsync(string blobName, System.BinaryData data)
         {
-            var containerClient = _blobServiceClient.GetBlobContainerClient("getmessage-reply-blobstorage");
+            var containerClient = _blobServiceClient.GetBlobContainerClient(Environment.GetEnvironmentVariable("BlobStorageContainerName"));
```
Comment (xnetsrak) [inline samples/SubDomain/GetMessage/Storage/StorageController.cs:32]: Added environment variables to local.settings.sample with empty value, then other developers know it's required.

**Review by xnetsrak — CHANGES_REQUESTED** [review]

**Review by xnetsrak — APPROVED** [review]

**Review by michaelmajgaard — APPROVED** [review]

**Pull request merged**